### PR TITLE
[Dashboard] Remove AddressesModal

### PR DESCRIPTION
**Pull request by kien-ngo**
Internal disucssion: https://thirdwebdev.slack.com/archives/C04PAQYN5HB/p1727426894785319

### CODE REVIEW

**Comment by kien-ngo** [comment]
* **#4825** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4825?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/components/contract-components/published-contract/index.tsx [hunk 1]
```diff
@@ -235,23 +234,6 @@ export const PublishedContract: React.FC<PublishedContractProps> = ({
                             : "Proxy"}{" "}
                           Enabled
                         </Heading>
-                        {implementationAddresses &&
```
Comment (kien-ngo) [inline apps/dashboard/src/components/contract-components/published-contract/index.tsx:238]: Internal discussion:
https://thirdwebdev.slack.com/archives/C04PAQYN5HB/p1727426894785319

Comment (MananTank) [reply apps/dashboard/src/components/contract-components/published-contract/index.tsx:238]: @jnsdls  Can you confirm this as welll?

Comment (jnsdls) [reply apps/dashboard/src/components/contract-components/published-contract/index.tsx:238]: Should be fine, as Joaquim said we need to re-do it later but it’ll be a lot simpler UI wise

**Review by MananTank — APPROVED** [review]

**Pull request closed without merging**